R/add.segs.R

```diff
@@ -253,9 +253,9 @@ add.tree.segs <- function(
         y1 = tree.segs1$tipy,
         default.units = 'native',
         gp = gpar(
-            col = clone.out$v$edge.colour.1,
-            lwd = clone.out$v$edge.width.1,
-            lty = clone.out$v$edge.type.1
+            col = clone.out$v$edge.colour.1[match(tree.segs1$tip, clone.out$v$id)],
```

Comment:
Let's do this without calling `match` repeatedly. We'll save a lot of loops through the tree dataframe.